tests/unit/htdocs/Frameworks/textsanitizer/MyTextSanitizerExtendedTest.php

```
@@ -0,0 +1,106 @@
+
+#[CoversClass(MyTextSanitizerExtended::class)]
+class MyTextSanitizerExtendedTest extends TestCase
+{
+    private static bool $loaded = false;
+
+    public static function setUpBeforeClass(): void
+    {
+        if (!self::$loaded) {
+            // Initialize logger before any tests to avoid "risky" handler warnings
+            if (!isset($GLOBALS['xoopsLogger'])) {
+                $GLOBALS['xoopsLogger'] = \XoopsLogger::getInstance();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\XoopsLogger' in method 'setUpBeforeClass'. (undefined)

(StaticAccess)